doc/generate_cmake_rst.py

```diff
@@ -155,10 +161,26 @@ def generate_rst(files):
     parser = argparse.ArgumentParser(description='Crawls a path for .cmake files and extract documentation of functions and macros into reStructured text.')
     parser.add_argument('path', nargs='?', default='.', help='The path to be crawled')
     parser.add_argument('-o', '--output', help='The name of the generated rst file')
+    parser.add_argument('--skip_private', action="store_true", help='skip documented items not marked with @public')
+    parser.add_argument('--skip_undocumented', action="store_true", help='skip items without documentation')
```

Comment:
Please start the help with an upper case character to match the other help texts.